boot/zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,168 @@
+# CMakeLists.txt for building mcuboot as a Zephyr project
+#
+# Copyright (c) 2017 Open Source Foundries Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
do you need a cmake_miminum_version() here? the boilerplate from Zephy will do a check, but there might be something you're doing before that that it's not supported in older versions.

Reply:
That's not a recommended practice in the application development primer, but I can add one in if it's a good idea (though everything above is pretty simple list, set, etc.). I'm not using anything that isn't documented in the 3.8.2 CMake reference manual, so I'll use that to match Zephyr.